testsuite/classlibrary/TestCondition.sc

```diff
@@ -0,0 +1,16 @@
+TestCondition : UnitTest {
+
+	test_timeout {
+		var condition = Condition.new;
+		var failed = false;
+		var lateTask = fork {
+			0.12.wait;
+			failed = true;
+			condition.unhang;
+		};
+		condition.hang(0.1);
```

Comment:
please reduce this to something small, `1e-6` should be fine. all of these small 'brief waits' in tests will add up to a lot of time if we don't pay attention to this.